test/clangd_tests.vim

```diff
@@ -1356,7 +1356,7 @@ def g:Test_LspTagFunc()
   END
   writefile(lines, 'Xtagfunc.c')
   :silent! edit Xtagfunc.c
-  g:WaitForServerFileLoad(1)
+  g:WaitForServerFileLoad(3)
```

Comment:
I don't see the diag test failure with the latest version of the plugin and Vim.  The tests for other recent merges are passing without this failure.  Can you revert this change?